### DISCUSSION TITLE
Process completed with exit code 2

### DISCUSSION THREAD

**Discussion by JankleDev** [community/community#108610] [New to GitHub]
Can anyone help me
https://github.com/JankleDev/AndroidPHP/actions/runs/7941179609/job/21683252362#step:9:23
Is failing everytime.

**Comment by rafidRD** [comment]
For OPcache problem: [here](https://help.nextcloud.com/t/the-php-opcache-is-not-properly-configured/36147)
Also try checking the specific version and documentation before using.
